react-app-todo/.eslintrc.cjs

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended',
+  ],
+  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
+  settings: { react: { version: '18.2' } },
+  plugins: ['react-refresh'],
+  rules: {
+    'react-refresh/only-export-components': 'warn',
+  },
+}
```

Comment:
[OPTIONAL] I think it would be better that you should only need to push the files inside the react-app-todo folder. You don't need to push that folder too.

Reply:
cetrtianly, I will do that!